tests/FlatBuffers.Test.Swift/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
```

Comment:
How does this relate to the same dirs ignored in the root?
Why is this dir called not just `swift` ?

Reply:
will be removing this gitignore